docs/blog/rendered/campaign-finance.ipynb

```diff
@@ -0,0 +1,1903 @@
+{
```

Comment:
<p>Should this be uncommented?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>oops, yes.</p>